src/fl/server.py

```diff
@@ -1,12 +1,49 @@
 from collections import namedtuple
 import logging
 import os
-from omegaconf import OmegaConf
+from omegaconf import OmegaConf, DictConfig
 import mlflow
 from socket import gethostname
 from flwr.server import start_server
+from flwr.server.strategy import FedAvg
+from federation.strategy import MCFedAvg, MCFedAdagrad, MCFedAdam, MCQFedAvg, fit_round
 
-from federation.strategy import MlflowStrategy, fit_round
+
+def get_strategy(strategy_params: DictConfig, checkpoint_dir: str) -> FedAvg:
+    """Creates flwr Strategy from strategy config entry
+
+    Args:
+        strategy_params (DictConfig): Strategy params with name, node_count and optional args dict
+        checkpoint_dir (str): Directory for saving model checkpoints
+
+    Raises:
+        ValueError: If strategy_params.name is unknown
+
+    Returns:
+        FedAvg: flwr Strategy with checkpointing and mlflow logging capabilities
+    """    
+    default_args = OmegaConf.create({
+        "fraction_fit": 0.99,
```

Comment:
what is fraction_fit and fraction_eval? a minimum fraction of clients available to fit and evaluate?

Reply:
Yes, and in our case we want every client to always be available for fit and evaluate